src/Rebing/GraphQL/Support/SelectFields.php

```diff
@@ -115,7 +116,12 @@ protected static function handleFields(array $requestedFields, $parentType, arra
                 continue;
             }
 
-            $fieldObject = $parentType->getField($key);
+            // If field doesn't exist on definition we don't select it
+            try {
+                $fieldObject = $parentType->getField($key);
+            } catch (\Exception $e) {
```

Comment:
I don't think this the correct solution. Maybe check, if the field exists or catch the specific exception?